konveyor/pelorus
# Refactor bitbucket collector to not swallow errors & have nicer logging

resolves #464

## Testing Instructions

Positive test:

1. Make a pelorus repo on bitbucket
2. Make an app password in bitbucket
3. Deploy a bitbucket committime collector using that bitbucket repo and credentials
4. Confirm that the metrics are logged / collected correctly

SSL Error test:
1. Keep the existing repo and deployment
2. Set up mockoon, enable TLS
3. Locally change the code so `git_server` is always overwritten with the local mockoon url
4. Verify the TLS/SSL error in the logs

Now, the negative test:

0. Remove the hardcoded git_server from the above test
1. Delete that repo
2. Redeploy the pod (don't rebuild!)
3. Confirm that the logs report the error and are fully detailed.
